go.mod

```diff
@@ -4,6 +4,8 @@ go 1.13
 
 // Pin k8s.io/* dependencies to kubernetes-1.16.0 to match controller-runtime v0.4.0
 replace (
+	// This must match the version below
+	go.etcd.io/etcd => go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
```

Comment:
Actually, this doesn't _have_ to match the version below. `replace` directives are always used regardless of the `require` directives below IF you are building the current module.

If something imports this module, then the `replace` directives are ignored, at which point those people depending on us may need a similar `replace` directive in their own module.

Reply:
Would you say it's 'good practice' to match as below?